### PR TITLE
fix: application edit mobile view

### PR DESCRIPTION
**Pull request by imchell**
Before:
before 1
before 2
After:
after 1
after 2

### CODE REVIEW

File: web/src/ApplicationEditPage.js [hunk 1]
```diff
@@ -720,40 +720,87 @@ class ApplicationEditPage extends React.Component {
...
-          >
-            {i18next.t("application:Copy signin page URL")}
-          </Button>
-          <br />
-          <div style={{position: "relative", width: "90%", border: "1px solid rgb(217,217,217)", boxShadow: "10px 10px 5px #888888", alignItems: "center", overflow: "auto", flexDirection: "column", flex: "auto"}}>
-            <LoginPage type={"login"} mode={"signin"} application={this.state.application} />
-            <div style={maskStyle} />
-          </div>
-        </Col>
+        {Setting.isMobile() ?
```
Comment (hsluoyz) [inline web/src/ApplicationEditPage.js:723]: Can we remove the code duplicates as much as possible?

**Review by hsluoyz — CHANGES_REQUESTED** [review]

**Comment by hsluoyz** [comment]
@imchell use English in screenshots

File: web/src/ApplicationEditPage.js (hunk 1 above)
Comment (imchell) [reply web/src/ApplicationEditPage.js:788]: Removed it

File: web/src/ApplicationEditPage.js [hunk 2]
```diff
@@ -710,72 +714,117 @@ class ApplicationEditPage extends React.Component {
...
-        </Button>
-        <br />
-        <div style={{position: "relative", width: "90%", border: "1px solid rgb(217,217,217)", boxShadow: "10px 10px 5px #888888", flexDirection: "column", flex: "auto"}}>
-          <PromptPage application={this.state.application} account={this.props.account} />
-          <div style={maskStyle} />
-        </div>
-      </Col>
+      <>
+        {Setting.isMobile() ?
+          (<Col span={22}>
```
Comment (hsluoyz) [inline web/src/ApplicationEditPage.js:820]: Use ```<Col span={Setting.isMobile() ? 22 : 11}>```, so `renderPrompt` is not needed

File: web/src/ApplicationEditPage.js [hunk 3]
```diff
@@ -710,72 +714,117 @@ class ApplicationEditPage extends React.Component {
...
-          <br />
-          <div style={{position: "relative", width: "90%", border: "1px solid rgb(217,217,217)", boxShadow: "10px 10px 5px #888888", alignItems: "center", overflow: "auto", flexDirection: "column", flex: "auto"}}>
-            <LoginPage type={"login"} mode={"signin"} application={this.state.application} />
-            <div style={maskStyle} />
-          </div>
-        </Col>
+        {Setting.isMobile() ?
+          (
+            <>
+              <Row gutter={[16, 24]}>
```
Comment (hsluoyz) [inline web/src/ApplicationEditPage.js:791]: Is `<Row gutter={[16, 24]}>` necessary?

**Review by hsluoyz — CHANGES_REQUESTED** [review]

**Review by hsluoyz — APPROVED** [review]

**Pull request merged**